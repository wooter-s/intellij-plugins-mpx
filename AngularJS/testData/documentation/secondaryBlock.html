@if(foo) {

} @else <caret>if (bar) {

}