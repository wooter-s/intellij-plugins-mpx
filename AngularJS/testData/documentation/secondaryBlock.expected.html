<div class='definition'><pre><span style='color: #000080'>block</span> <span style='color: #000000'>@else if</span></pre></div>
<div class='content'>
<p>The <code>@else if</code> block conditionally displays its content when its condition expression is truthy and all previous <code>@if</code> and <code>@if else</code> conditions were falsy.</p>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Primary block:</td><td valign='top'><span style='color: #000080'>@if</span></td>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>@angular/core@17.0.0-rc.0</td>
</table>